pages/login/index.tsx

```diff
@@ -0,0 +1,8 @@
+import Login from "@/components/Login";
+import React from "react";
+
+const LoginPage = () => {
```

Comment:
See these files can hold a lot of server side logic inside `getServerSideProps` - utilise that